src/items.zig

```diff
@@ -156,6 +156,53 @@ const MaterialProperty = enum {
 	}
 };
 
+pub const BaseItemIndex = packed struct {
+	index: u16,
+
+	pub fn fromId(_id: []const u8) ?BaseItemIndex {
+		return reverseIndices.get(_id);
+	}
+	pub fn baseItem(self: BaseItemIndex) *BaseItem {
+		return &itemList[self.index];
+	}
+	pub fn image(self: BaseItemIndex) graphics.Image {
+		return itemList[self.index].image;
+	}
+	pub fn texture(self: BaseItemIndex) ?graphics.Texture {
+		return itemList[self.index].getTexture();
```

Comment:
Should be just `texture` to keep the old behavior, if that's the intention here.

Reply:
Done.